### PR TITLE
Update AZ400_M00 and AZ400_M01_L01

### PR DESCRIPTION
**Pull request by WigF1**
**Replace issue name LAB00:QUICK_DESCRIPTION, for example "LAB01: My new issue" (or same name as linked Issue)**

## Related Issue

**Link related Github Issue** 🢂 Fixes # . (Include issue number after #)

## Checklist 
Mark completed with "x" between brackets, "[x]", or checking the box once the PR is created:
- [x] Has related GitHub Issue 💥 [Create Issue](https://github.com/MicrosoftLearning/AZ400-DesigningandImplementingMicrosoftDevOpsSolutions/blob/master/.github/CONTRIBUTING.md#reporting-issues) 📝
- [x] Tested it
- [x] Read the PR collaboration guide 👓 [Collaboration Guide](https://github.com/MicrosoftLearning/AZ400-DesigningandImplementingMicrosoftDevOpsSolutions/blob/master/.github/CONTRIBUTING.md#pull-requests) 📝

See #573 for the issue.

### CODE REVIEW

File: Instructions/Labs/AZ400_M01_L01_Agile_Plan_and_Portfolio_Management_with_Azure_Boards.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ After you complete this lab, you will be able to:
 - Define dashboards.
 - Customize team process.
 
-## Estimated timing: 60 minutes
+## Estimated timing: 90 minutes
```
Comment (rob-foulkrod) [inline Instructions/Labs/AZ400_M01_L01_Agile_Plan_and_Portfolio_Management_with_Azure_Boards.md:32]: Lab timings are a separate conversation. Please revert.

File: Instructions/Labs/AZ400_M01_L01_Agile_Plan_and_Portfolio_Management_with_Azure_Boards.md [hunk 2]
```diff
@@ -349,9 +349,9 @@ To maximize a team's ability to consistently deliver high quality software, Kanb
     ![On the "Settings" panel, clikc "Save & Close"](images/m1/EShop-WEB-qa_column_v1.png)
 
 1. On the **Settings** panel, select the **Columns** tab again. Notice the ellipsis next to the **QA Approved** column you created. Select **Move right** twice, so that the QA Approved column gets positioned in-between **Committed** and **Done**.
-1. On the **Settings** panel, click **Save**.
+1. On the **Settings** panel, click **Save amd close**.
```
Comment (rob-foulkrod) [inline Instructions/Labs/AZ400_M01_L01_Agile_Plan_and_Portfolio_Management_with_Azure_Boards.md:352]: Correct Spelling

**Comment by rob-foulkrod** [comment]
Thanks for the contribution. Please be patient while we find time to review.

File: Instructions/Labs/AZ400_M01_L01_Agile_Plan_and_Portfolio_Management_with_Azure_Boards.md [hunk 3]
```diff
@@ -198,7 +198,7 @@ Work items play a prominent role in Azure DevOps. Whether describing work to be
 
     > **Note**: You can use the second plus sign directly left to the first backlog item to add a new task to it.
 
-1. On the **Backlog** tab of the **EShop-WEB** pane, in the upper left corner of the pane, click the second plus sign from the top, the one next to the first work item. This will display the **NEW TASK** panel.
+1. On the **Backlog** tab of the **EShop-WEB** pane, in the upper left corner of the pane, click the plus sign next to the first work item. This will display the **NEW TASK** panel.
```
Comment (yashints) [inline Instructions/Labs/AZ400_M01_L01_Agile_Plan_and_Portfolio_Management_with_Azure_Boards.md:201]: why is the second plus sign removed here?

File: Instructions/Labs/AZ400_M01_L01_Agile_Plan_and_Portfolio_Management_with_Azure_Boards.md [hunk 4]
```diff
@@ -228,15 +228,15 @@ The sprint backlog should contain all the information the team needs to successf
 
     > **Note**: The current sprint has a pretty limited scope. There are two tasks in the **To do** stage. At this point, neither task has been assigned. Both show a numeric value to the right of **Unassigned** entry representing the remaining work estimate.
 
-1. Within the **ToDo** Column, notice the Task Item **Add page for most recent tutorials**, click the **Unassigned** entry and, in the list of user accounts, select your account to assign the task to yourself.
+1. Within the **ToDo** Column, notice the Task Item **Add page for most recent tutorials**, replace the **Unassigned** entry with your user account, then select your account to assign the task to yourself.
```
Comment (yashints) [inline Instructions/Labs/AZ400_M01_L01_Agile_Plan_and_Portfolio_Management_with_Azure_Boards.md:231]: revert back please, this version is confusing

File: Instructions/Labs/AZ400_M01_L01_Agile_Plan_and_Portfolio_Management_with_Azure_Boards.md (hunk 2 above)
Comment (yashints) [inline Instructions/Labs/AZ400_M01_L01_Agile_Plan_and_Portfolio_Management_with_Azure_Boards.md:352]: typo

File: Instructions/Labs/AZ400_M01_L01_Agile_Plan_and_Portfolio_Management_with_Azure_Boards.md [hunk 5]
```diff
@@ -349,9 +349,9 @@ To maximize a team's ability to consistently deliver high quality software, Kanb
     ![On the "Settings" panel, clikc "Save & Close"](images/m1/EShop-WEB-qa_column_v1.png)
 
 1. On the **Settings** panel, select the **Columns** tab again. Notice the ellipsis next to the **QA Approved** column you created. Select **Move right** twice, so that the QA Approved column gets positioned in-between **Committed** and **Done**.
-1. On the **Settings** panel, click **Save**.
+1. On the **Settings** panel, click **Save amd close**.
 
-1. **Refresh** the **Boards portal**, so the **QA Approved** column is visible in the Kanban board view now.
+1. On the **Boards portal**, the **QA Approved** column is now visible in the Kanban board view.
```
Comment (yashints) [inline Instructions/Labs/AZ400_M01_L01_Agile_Plan_and_Portfolio_Management_with_Azure_Boards.md:354]: revert back please, notes are rarely read by learners

File: Instructions/Labs/AZ400_M01_L01_Agile_Plan_and_Portfolio_Management_with_Azure_Boards.md [hunk 6]
```diff
@@ -374,7 +374,7 @@ To maximize a team's ability to consistently deliver high quality software, Kanb
 
     ![On the "Settings" panel, review information and click "Save and close"](images/m1/dd_v1.png)
 
-    > **Note**: The **QA Approved** stage now has **Doing** and **Done** columns. You can also click the informational symbol (with letter **i** in a circle) next to the column header to read the **Definition of done**.
+    > **Note**: The **QA Approved** stage now has **Doing** and **Done** columns. You can also click the informational symbol (with letter **i** in a circle) next to the column header to read the **Definition of done**. You may need to refresh the browser to see changes.
```
Comment (yashints) [inline Instructions/Labs/AZ400_M01_L01_Agile_Plan_and_Portfolio_Management_with_Azure_Boards.md:377]: see above

File: Instructions/Labs/AZ400_M01_L01_Agile_Plan_and_Portfolio_Management_with_Azure_Boards.md [hunk 7]
```diff
@@ -384,12 +384,14 @@ To maximize a team's ability to consistently deliver high quality software, Kanb
...
 1. On the **Settings** panel, select the **Swimlanes** tab.
 1. On the **Swimlanes** tab, click **+ Swimlane**, directly under the **Swimlane name** label, in the **Name** textbox, type **Expedite**.
-1. On the **Settings** panel, click **Save**.
+1. On the **Settings** panel, click **Save and close**.
 
     ![On the "Settings" panel, review information and click "Save"](images/m1/EShop-WEB-swimlane_v1.png)
 
 1. Back on the **Board** tab of the **Boards** panel, drag and drop the **Committed** work item onto the **QA Approved \| Doing** stage of the **Expedite** swimlane so that it gets recognized as having priority when QA bandwidth becomes available.
 
+    > **Note**: You may need to refresh the browser to make the swimlane visible.
```
Comment (yashints) [inline Instructions/Labs/AZ400_M01_L01_Agile_Plan_and_Portfolio_Management_with_Azure_Boards.md:393]: revert back please, they appear immediately

File: Instructions/Labs/AZ400_M01_L01_Agile_Plan_and_Portfolio_Management_with_Azure_Boards.md [hunk 8]
```diff
@@ -460,7 +462,7 @@ Dashboards allow teams to visualize status and monitor progress across the proje
 
     ![If necessary, on the "Dashboards" pane, in the upper left corner, in the "eShopOnWeb Team" section, select "eShopOnWeb Team - Overview"](images/m1/EShop-WEB-dashboard_v1.png)
 
-1. On the **Dashboards** pane, select the drop-down menu next to the **eShopOnWeb Team - Overview** title, and select **+ New dashboard**.
+1. On the **Dashboards** pane, in the upper-right corner, select **+ New dashboard**.
```
Comment (yashints) [inline Instructions/Labs/AZ400_M01_L01_Agile_Plan_and_Portfolio_Management_with_Azure_Boards.md:465]: The new layout is slightly different, if you have time update the instructions and the image (now it's a drop down in the left side)

File: Instructions/Labs/images/m1/EShop-WEB-pbi_field_layout_v1.png
Comment (yashints) [inline Instructions/Labs/images/m1/EShop-WEB-pbi_field_layout_v1.png:1]: Is this intentional? I think you saved the image with the wrong name?

**Review by yashints — CHANGES_REQUESTED** [review]
Thanks for your time and contribution

File: Instructions/Labs/AZ400_M00_Validate_lab_environment.md [hunk 9]
```diff
@@ -32,10 +32,14 @@ lab:
 1. Once the screen shows the linked Azure Subscription ID at the top, change the number of **Paid parallel jobs** for **MS Hosted CI/CD** from 0 to **1**. Then click the **SAVE** button at the bottom.
 1. In **Organization Settings**, go to section **Pipelines** and click **Settings**.
 1. Toggle the switch to **Off** for **Disable creation of classic build pipelines** and **Disable creation of classic release pipelines**
+
     > Note: The **Disable creation of classic release pipelines** switch sets to **On** hides classic release pipeline creation options such as the **Release** menu in the **Pipeline** section of DevOps projects.
+
```
[the remark this answers is not in the document]
Comment (WigF1) [reply Instructions/Labs/AZ400_M00_Validate_lab_environment.md:37]: Without the extra line in there, the note is not displayed correctly:

image

File: Instructions/Labs/AZ400_M00_Validate_lab_environment.md [hunk 10]
```diff
@@ -32,10 +32,14 @@ lab:
 1. Once the screen shows the linked Azure Subscription ID at the top, change the number of **Paid parallel jobs** for **MS Hosted CI/CD** from 0 to **1**. Then click the **SAVE** button at the bottom.
 1. In **Organization Settings**, go to section **Pipelines** and click **Settings**.
 1. Toggle the switch to **Off** for **Disable creation of classic build pipelines** and **Disable creation of classic release pipelines**
+
```
[the remark this answers is not in the document]
Comment (WigF1) [reply Instructions/Labs/AZ400_M00_Validate_lab_environment.md:35]: Without the extra line in there, the note is not displayed correctly:

image

File: Instructions/Labs/AZ400_M01_L01_Agile_Plan_and_Portfolio_Management_with_Azure_Boards.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (WigF1) [reply Instructions/Labs/AZ400_M01_L01_Agile_Plan_and_Portfolio_Management_with_Azure_Boards.md:32]: OK, I will revert it, however, at no point have I ever encountered a student complete this lab in 60 minutes. You need to engage with your MCTs that deliver this content and ask them directly.

File: Instructions/Labs/AZ400_M01_L01_Agile_Plan_and_Portfolio_Management_with_Azure_Boards.md (hunk 7 above)
Comment (WigF1) [reply Instructions/Labs/AZ400_M01_L01_Agile_Plan_and_Portfolio_Management_with_Azure_Boards.md:393]: They do not - I literally did this yesterday as I was making changes. The swimlanes did NOT appear.

File: Instructions/Labs/AZ400_M01_L01_Agile_Plan_and_Portfolio_Management_with_Azure_Boards.md (hunk 5 above)
Comment (WigF1) [reply Instructions/Labs/AZ400_M01_L01_Agile_Plan_and_Portfolio_Management_with_Azure_Boards.md:354]: So why are they so copious in the labs?

File: Instructions/Labs/AZ400_M01_L01_Agile_Plan_and_Portfolio_Management_with_Azure_Boards.md (hunk 6 above)
Comment (WigF1) [reply Instructions/Labs/AZ400_M01_L01_Agile_Plan_and_Portfolio_Management_with_Azure_Boards.md:377]: See above

File: Instructions/Labs/AZ400_M00_Validate_lab_environment.md [hunk 11]
```diff
@@ -32,10 +32,14 @@ lab:
...
     > Note: The **Disable creation of classic release pipelines** switch sets to **On** hides classic release pipeline creation options such as the **Release** menu in the **Pipeline** section of DevOps projects.
+
 1. In **Organization Settings**, go to section **Security** and click **Policies**.
 1. Toggle the switch to **On** for **Allow public projects**
+
```
[the remark this answers is not in the document]
Comment (WigF1) [reply Instructions/Labs/AZ400_M00_Validate_lab_environment.md:40]: Without the extra line, the note is not rendered correctly. All other notes have this.

image

File: Instructions/Labs/AZ400_M00_Validate_lab_environment.md [hunk 12]
```diff
@@ -32,10 +32,14 @@ lab:
...
     > Note: Extensions used in some labs might require a public project to allow using the free version.
+
```
[the remark this answers is not in the document]
Comment (WigF1) [reply Instructions/Labs/AZ400_M00_Validate_lab_environment.md:42]: Without the extra line, the note is not rendered correctly. All other notes have this.

image

File: Instructions/Labs/AZ400_M01_L01_Agile_Plan_and_Portfolio_Management_with_Azure_Boards.md (hunk 1 above)
Comment (WigF1) [reply Instructions/Labs/AZ400_M01_L01_Agile_Plan_and_Portfolio_Management_with_Azure_Boards.md:32]: Roger

File: Instructions/Labs/AZ400_M01_L01_Agile_Plan_and_Portfolio_Management_with_Azure_Boards.md (hunk 2 above)
Comment (WigF1) [reply Instructions/Labs/AZ400_M01_L01_Agile_Plan_and_Portfolio_Management_with_Azure_Boards.md:352]: Corrected

File: Instructions/Labs/images/m1/EShop-WEB-pbi_field_layout_v1.png
Comment (WigF1) [reply Instructions/Labs/images/m1/EShop-WEB-pbi_field_layout_v1.png:1]: Nope.

When the lab steps are followed as written, that image is exactly what students see. If they were instructed to add a field and click 'Add field' and then click 'New group', they would see the image that was in the instructions. However, that's not what the instructions say. The grouping is defined at the same time as the field.

File: Instructions/Labs/AZ400_M01_L01_Agile_Plan_and_Portfolio_Management_with_Azure_Boards.md (hunk 7 above)
Comment (WigF1) [reply Instructions/Labs/AZ400_M01_L01_Agile_Plan_and_Portfolio_Management_with_Azure_Boards.md:393]: I just repeated the steps a few more times. Out of 5 times, the swimlanes appeared automatically only once. Is it not possible to leave the note in there? At least when they don't appear, students will know it's simply a refresh?

Students are instructed to do so for the processes earlier in the lab, how is this different?

File: Instructions/Labs/AZ400_M01_L01_Agile_Plan_and_Portfolio_Management_with_Azure_Boards.md (hunk 3 above)
Comment (WigF1) [reply Instructions/Labs/AZ400_M01_L01_Agile_Plan_and_Portfolio_Management_with_Azure_Boards.md:201]: Because the image is incorrect when the lab steps are followed as written. As is indicated by the screenshots included in the lab steps.

If you look at the original image at images/m1/new_task_v1.png, it does not match the ordering of items from the prior image (eshop-web-view_backlog_v1.png) nor does it match the correct order displayed in EShop-WEB-over_capacity_v1.png.

I assume some form of sort had been placed when screenshots were taken. 

In order to achieve the desired result, the task needs adding to the first pbi, not the second.

File: Instructions/Labs/AZ400_M01_L01_Agile_Plan_and_Portfolio_Management_with_Azure_Boards.md (hunk 4 above)
Comment (WigF1) [reply Instructions/Labs/AZ400_M01_L01_Agile_Plan_and_Portfolio_Management_with_Azure_Boards.md:231]: There is no list of user accounts displayed when clicking 'Unassigned'. As written, the lab steps do not work.

File: Instructions/Labs/AZ400_M00_Validate_lab_environment.md (hunk 9 above)
[the remark this answers is not in the document]
Comment (yashints) [reply Instructions/Labs/AZ400_M00_Validate_lab_environment.md:37]: Where are you seeing this? is that in the lab environment or a markdown editor like VS Code?

File: Instructions/Labs/AZ400_M01_L01_Agile_Plan_and_Portfolio_Management_with_Azure_Boards.md (hunk 5 above)
Comment (yashints) [reply Instructions/Labs/AZ400_M01_L01_Agile_Plan_and_Portfolio_Management_with_Azure_Boards.md:354]: I don't get your point, having the refresh in instructions is better than notes no?

File: Instructions/Labs/AZ400_M01_L01_Agile_Plan_and_Portfolio_Management_with_Azure_Boards.md (hunk 4 above)
Comment (yashints) [reply Instructions/Labs/AZ400_M01_L01_Agile_Plan_and_Portfolio_Management_with_Azure_Boards.md:231]: As you correctly mentioned there won't be a user account to select, so the user has to type their name and press enter to see the account. Maybe reword to this instead

File: Instructions/Labs/AZ400_M01_L01_Agile_Plan_and_Portfolio_Management_with_Azure_Boards.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (rob-foulkrod) [reply Instructions/Labs/AZ400_M01_L01_Agile_Plan_and_Portfolio_Management_with_Azure_Boards.md]: So you know who you are working with, both Yas and I are MCTs who deliver this course on the regular. We are helping out the content creators in our extra time.

File: Instructions/Labs/AZ400_M01_L01_Agile_Plan_and_Portfolio_Management_with_Azure_Boards.md (hunk 6 above)
Comment (rob-foulkrod) [reply Instructions/Labs/AZ400_M01_L01_Agile_Plan_and_Portfolio_Management_with_Azure_Boards.md:377]: Currently, contributions that correct errors are the most helpful. Edits to style are more arbitrary and debatable and result in unnecessary churn. Keeping the PR smaller and more focused will help us merge it more quickly.

**Review by LuizMacedo — APPROVED** [review]
